lib/format.ex

```diff
@@ -15,7 +15,7 @@ defmodule Momento.Format do
       "7-1-16"
   """
 
-  @tokens ~r/YYYY|YY?|MM?M?M?|DD?D?D?|HH?|mm?|ss?|X|x/
+  @tokens ~r/YYYY|YY?|MMMM|MMM|MM?M?M?|Do|DD?D?D?|HH?|mm?|ss?|X|x/
```

Comment:
You don't need to add `MMMM` and `MMM` because it is handled by `MM?M?M?`. The `?` in regex means that the previous character is optional. So, `MM?M?M?` will match all `MMMM`, `MMM`, `MM` and `M`.